rules/windows/execution_command_shell_via_rundll32.toml

```diff
@@ -0,0 +1,53 @@
+[metadata]
+creation_date = "2020/10/19"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/10/19"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies command shell activity started via rundll32 which is usually abused by attackers to host malicious code."
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Command shell activity started via rundll32"
+risk_score = 21
+rule_id = "9ccf3ce0-0057-440a-91f5-870c6ad39093"
+severity = "low"
+tags = ["Elastic", "Windows"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+ wildcard(process.name, "cmd.exe", "powershell.exe") and
```

Comment:
I've tried something like this before but it was noisy.
Any ideas to make this higher fidelity? If not, that's fine, we can capture its noise level with metadata.

Reply:
I've used this before as a hunt in prod, usually main FP is related to rundll32 with cmdline value like "*\\SHELL32.dll*RunAsNewUser_RunDLL*" (result of right-click runas new user of cmd), few other exception is when EDR does not capture parent commanding (e.g. NA or NULL). As you suggested we can play and tune it with other attributes once we have alert telemetr.

Reply:
i would expect a lot more results matching the rundll32 -> cmd chain

Reply:
do you have example of mayches other than the RunAsNewUser_RunDLL one ?, used this before in prod with more than 3000 endpoints and that was the only main match (90% of cases). given the fact that CS and other backdoors default to rundll32 and their cmd|interactive shell command defaults to cmd.exe its worth alerting (ratio viz vs noise is very acceptable).